### PR TITLE
Added tildification to uwuify

### PR DESCRIPTION
**Pull request by SomeHybrid**

### CODE REVIEW

**Comment by SomeHybrid** [comment]
um 

yes

File: src/imsosorry/uwuification.py [hunk 1]
```diff
@@ -107,12 +109,28 @@ def emoji_replace(match: re.Match, strength: float = 0.0) -> str:
...
     text = word_replace(text)
     text = nyaify(text)
     text = char_replace(text)
     text = stutter(text, stutter_strength)
     text = emoji(text, emoji_strength)
+    text = tildify(text, tilde_strength)
     return text
+
+
+print(uwuify("Hello, world!"))
```
Comment (shenanigansd) [inline src/imsosorry/uwuification.py:136]: Please don't leave any global side effects.

File: src/imsosorry/uwuification.py [hunk 2]
```diff
@@ -107,12 +109,28 @@ def emoji_replace(match: re.Match, strength: float = 0.0) -> str:
...
+        return "~"
+    return match_string
+
+
+def tildify(text: str, strength: float) -> str:
+    """Adds some tildes to spaces."""
+    return REGEX_TILDE.sub(partial(tildes, strength=strength), text, 0)
+
+
+def uwuify(text: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1, tilde_strength: float = 1) -> str:
```
Comment (shenanigansd) [inline src/imsosorry/uwuification.py:124]: Don't just max it out, the randomness is part of the fun!

**Review by shenanigansd — COMMENTED** [review]
Thanks for the contribution!
Everything looks to be in order, just a couple of quick comments. If you'll take care of those and the linter error, I'll go ahead and merge this.

**Comment by SomeHybrid** [comment]
Ah sorry those were some tests i forgot to remove :p

File: src/imsosorry/uwuification.py (hunk 1 above)
Comment (shenanigansd) [reply src/imsosorry/uwuification.py]: 18b610f216b024f70447373c60c5e504f3e467f4

File: src/imsosorry/uwuification.py (hunk 2 above)
Comment (shenanigansd) [reply src/imsosorry/uwuification.py]: 18b610f216b024f70447373c60c5e504f3e467f4

File: src/imsosorry/uwuification.py [hunk 3]
```diff
@@ -107,12 +109,25 @@ def emoji_replace(match: re.Match, strength: float = 0.0) -> str:
...
+        return "~"
+    return match_string
+
+
+def tildify(text: str, strength: float) -> str:
+    """Adds some tildes to spaces."""
+    return REGEX_TILDE.sub(partial(tildes, strength=strength), text, 0)
+
+
+def uwuify(text: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1, tilde_strength: float = 0.1) -> str:
```
Comment (shenanigansd) [inline src/imsosorry/uwuification.py:124]: This line is now too long for `black`
```suggestion
def uwuify(
    text: str,
    *,
    stutter_strength: float = 0.2,
    emoji_strength: float = 0.1,
    tilde_strength: float = 0.1,
) -> str:
```

File: src/imsosorry/uwuification.py [hunk 4]
```diff
@@ -107,12 +109,25 @@ def emoji_replace(match: re.Match, strength: float = 0.0) -> str:
     return match_string
 
 
-def uwuify(text: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1) -> str:
+def tildes(match: re.Match, strength: float = 0.0):
```
Comment (shenanigansd) [inline src/imsosorry/uwuification.py:112]: All methods need docstrings
```suggestion
def tildes(match: re.Match, strength: float = 0.0):
"""Adds some tildes to spaces."""
```

File: src/imsosorry/uwuification.py (hunk 3 above)
Comment (shenanigansd) [inline src/imsosorry/uwuification.py:124]: This line is now too long for `black`
```suggestion
def uwuify(
    text: str,
    *,
    stutter_strength: float = 0.2,
    emoji_strength: float = 0.1,
    tilde_strength: float = 0.1,
) -> str:
```

**Review by shenanigansd — COMMENTED** [review]
Looks good, just take care of the lints and we'll merge this

File: src/imsosorry/uwuification.py (hunk 4 above)
Comment (shenanigansd) [reply src/imsosorry/uwuification.py:112]: 3f65f287a53a96bfca8d1b8700577c730927f688

File: src/imsosorry/uwuification.py (hunk 3 above)
Comment (shenanigansd) [reply src/imsosorry/uwuification.py]: 3f65f287a53a96bfca8d1b8700577c730927f688

**Pull request merged**